foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/actions/spi/ActionMetadata.java

```diff
@@ -28,6 +29,7 @@
     private final Class<? extends Action> action;
     private final boolean cacheable;
     private final String cacheToken;
+    private EmailClient emailClient;
```

Comment:
This is metadata. Why is EmailClient being added here?

Reply:
This is removed